tests/kafkatest/version/version.py

```diff
@@ -1,17 +1,19 @@
-# Copyright 2015 Confluent Inc.
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I also thought rat checked this - not sure how this slipped through the first time